v3/tutorials/07-add-a-pallet/index.mdx

```diff
@@ -631,10 +647,12 @@ the beginning of this guide, each pallet will be a little different. Have no fea
 refer to the [Substrate runtime](https://github.com/paritytech/substrate/tree/master/bin/node/runtime)
 which includes nearly every pallets in the library of core FRAME pallets.
 
-In the Substrate node runtime `bin/node/runtime/Cargo.toml` file, you will see how to import
-different pallets, and in the `bin/node/runtime/src/lib.rs` file you will see how to add each pallet to your
-runtime. You can generally copy what was done there as a starting point to include a pallet in your
-own runtime.
+In the Substrate node runtime
+[`bin/node/runtime/Cargo.toml` file](https://github.com/paritytech/substrate/blob/master/bin/node/runtime/Cargo.toml),
+you will see how to import different pallets, and in the
```

Comment:
Use monthly or latest tag might want to have a new tag we float on latest template for substrate itself, like `devhub-latest`

Reply:
When I tried the monthly tag, the tutorial wasn't successful. I was able to run through it all using the latest tag but I started with the tutorial in the old site, so it could have just been a mismatch between what I downloaded initially and when trying to re-do the tutorial using the instructions in the new substrate-docs repo.

Reply:
@NukeManDan I pushed a `devhub/latest` tag to Substrate repo. So ideally when pointing to rust docs, it would be `/rustdoc/latest/...` which really is the latest monthly, and when pointing to the src code it would be `substrate/tree/devhub/latest/...` which is the src code of latest monthly.

But I would suggest to keep it consistent and fix this across the whole docs after docs.substrate.io launch.

Reply:
@lsgunnlsgunn I re-compiled https://github.com/substrate-developer-hub/substrate-node-template/tree/tut-sols/add-a-pallet%2Flatest and found the issue. Fixed in the content (the `usize` -> `u32` data type fix) and the [code in node template](https://github.com/substrate-developer-hub/substrate-node-template/commit/b4bba5df0db6b4cff089ad021bdd5ee32d1ca670). Thanks.